eox_core/api/v1/tests/integration/test_views.py

```diff
@@ -16,6 +18,75 @@
 
 settings = ds.INTEGRATION_TEST_SETTINGS
 
+USER_URL = f"{settings['EOX_CORE_API_BASE']}{reverse('eox-api:eox-api:edxapp-user')}"
+USER_UPDATER_URL = f"{settings['EOX_CORE_API_BASE']}{reverse('eox-api:eox-api:edxapp-user-updater')}"
+ENROLLMENT_URL = f"{settings['EOX_CORE_API_BASE']}{reverse('eox-api:eox-api:edxapp-enrollment')}"
```

Comment:
This is not a blocker in any way. However, API URLs are usually defined inside each test suite setup method. Is there a specific reason for defining them here?

Reply:
I tried to define the URLs in each `SetUp` method but generated an error with the variables due to multiple inheritance. However, we can add these URLs in each Mixin: https://github.com/eduNEXT/eox-core/commit/a42c18622d9b24d566c7286f58a0a5ec887ff664

What do you think?

Reply:
That looks reasonable. Thank you!